pkg/kubectl/cmd/set/set_resources.go

```diff
@@ -111,7 +111,7 @@ func NewCmdResources(f cmdutil.Factory, out io.Writer, errOut io.Writer) *cobra.
 
 	cmdutil.AddPrinterFlags(cmd)
 	//usage := "Filename, directory, or URL to a file identifying the resource to get from the server"
-	//kubectl.AddJsonFilenameFlag(cmd, &options.Filenames, usage)
+	//kubectl.cmdutil.AddFilenameFlag(cmd, &options.Filenames, usage)
```

Comment:
nit: do we still want to keep the commented lines?

Reply:
Well spotted. Thanks.